webapp/src/pages/GameRulesPage/GameRulesPage.tsx

```diff
@@ -0,0 +1,148 @@
+import { gameQueueClientToServerWsEvents } from 'transcendence-shared';
+import {
+  ButtonProps,
+  ButtonVariant,
+  Header,
+  IconVariant,
+  ReactiveButton,
+  Text,
+  TextVariant,
+} from '../../shared/components';
+import { GamePairingStatusDtoGameQueueStatusEnum } from '../../shared/generated';
+import { useGamePairing } from '../../shared/hooks/UseGamePairing';
+import { useNavigation } from '../../shared/hooks/UseNavigation';
+import socket from '../../shared/socket';
+import { PLAY_GAME_TRAIN_URL } from '../../shared/urls';
+import './GameRulesPage.css';
+
+export default function GameRulesPage() {
+  const { goBack, navigate } = useNavigation();
+  const { gameQueueStatus } = useGamePairing();
+
+  let buttons = [
+    {
+      variant: ButtonVariant.SUBMIT,
+      iconVariant: IconVariant.PLAY,
+      onClick: () => navigate(PLAY_GAME_TRAIN_URL),
+      children: 'Training',
+    },
+  ];
+
+  if (gameQueueStatus === GamePairingStatusDtoGameQueueStatusEnum.None) {
+    buttons.push({
+      variant: ButtonVariant.SUBMIT,
+      iconVariant: IconVariant.ADD,
+      onClick: () => {
+        socket.emit(gameQueueClientToServerWsEvents.gameQueueJoin);
+      },
+      children: 'Join game queue',
+    });
+  }
+
+  return (
+    <div className="game-rules-page">
+      <Header icon={IconVariant.ARROW_BACK} onClick={goBack}>
+        game rules
+      </Header>
+      <div className="game-rules-page-main">
+        <div className="game-rules-page-main-objective">
+          <Text variant={TextVariant.SUBHEADING}>Objective:</Text>
+          <Text variant={TextVariant.PARAGRAPH}>
+            The objective of the game is to score points by hitting the ball
+            with the paddle and making it pass the opponent's paddle. The first
+            player to reach 10 points wins the game.
+          </Text>
+        </div>
+        <div className="game-rules-page-main-gameplay">
+          <Text variant={TextVariant.SUBHEADING}>Gameplay:</Text>
+          <ol>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                At the start of the game, both players need to click the "Ready"
+                button to indicate they are ready to play.
```

Comment:
This is obvious, I would remove it

Reply:
I think it's at least a good placeholder to put the configuration options as it is, when we will have them.

Reply:
I will leave this for now, but we could remove or update it in the future.